google/cloud/spanner_dbapi/connection.py

```diff
@@ -702,10 +698,6 @@ def set_autocommit_dml_mode(
         self._autocommit_dml_mode = autocommit_dml_mode
 
     def _partitioned_query_validation(self, partitioned_query, statement):
-        if _get_statement_type(Statement(partitioned_query)) is not StatementType.QUERY:
```

Comment:
Skip validation here and let Spanner return an error if the statement is not valid for `PartitionQuery`.